libs/libnx/nxfonts/.gitignore

```diff
@@ -1,2 +1,3 @@
 nxfonts_convert_*bpp.c
 nxfonts_bitmaps_*.c
+nxfonts_tom-thumb-4x6.c
```

Comment:
Don't since the generated name change to nxfonts_bitmaps_tom-thumb-4x6.c by commit:
```
commit 23389215bb09b2997c2e29a39b635339c202bf3e
Author: liuhaitao <liuhaitao@xiaomi.com>
Date:   Tue Apr 28 14:18:22 2020 +0800

    libnx/nxfonts: change the generated file nxfonts_tom-thumb-4x6.c to nxfonts_bitmap_tom-thumb-4x6.c
    
    Align the generated files name, so make distclean could also remove all nxfonts_bitmaps_*.c files, no
    need to add one more extra DELFILE for nxfonts_tom-thumb-4x6.c.
    
    Signed-off-by: liuhaitao <liuhaitao@xiaomi.com>
```
But forget to update .gitignore.

Reply:
Ah, okay, I missed that.  No need for this then, closing.